src/flask_container_scaffold/util.py

```diff
@@ -52,18 +52,18 @@ def _parse_cfg(config):
     return config_dict
 
 
-def parse_input(logger, obj, default_return=BaseApiModel):
+def parse_input(logger, obj, default_return=BaseApiView):
     """
     Parses incoming request, returns a serializable object to return
     to the client in all cases. When there is a failure, the
     object contains error information.
     :param Logger logger: Instantiated logger object
     :param BaseModel obj: An object type based on a pydantic BaseModel to
                           attempt to parse.
-    :param BaseApiModel default_return: An object type that will be returned if
-                                        validation of obj fails. This object
-                                        must descend from BaseApiModel or
-                                        implement an error field of type str.
+    :param BaseApiView default_return: An object type that will be returned if
+                                       validation of obj fails. This object
+                                       must descend from BaseApiView or
+                                       implement an errors field of type list.
```

Comment:
(question) Should 'list' say 'dict' here?

Reply:
+1

Reply:
Done!